app/helpers/admin/users_helper.rb

```diff
@@ -31,5 +31,19 @@ def format_role_tooltip(user)
         "Resource Admin: #{user.roles.pluck(:resource_type).compact.join(', ')}"
       end
     end
+
+    def user_status(user)
```

Comment:
We get the `user_status` in multiple places with a few tweaks. I suspect we can update this helper to make it re-useable in the future. I will add a ticket onto the board to look into this.